gatsby-browser.js

```diff
@@ -86,7 +86,9 @@ const pushwooshInit = (push_woosh) => {
     push_woosh.push([
         'onReady',
         function (api) {
-            push_woosh.subscribe()
+            if (!push_woosh.isSubscribed()){
```

Comment:
the function will return a promise, please chain it with then, and use the argument